src/applications/edu-benefits/10203/containers/IntroductionPage.jsx

```diff
@@ -5,9 +5,61 @@ import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
 import SaveInProgressIntro from '../content/SaveInProgressIntro';
 import { connect } from 'react-redux';
 
+import { getEnrollmentData } from '../../../post-911-gib-status/actions/post-911-gib-status';
+
 export class IntroductionPage extends React.Component {
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
+    this.props.getEnrollmentData().then(response =>
+      this.setState({
+        remainingEntitlement: response?.data?.remainingEntitlement,
+      }),
+    );
+  }
+
+  moreThanSixMonths = remaining => {
+    const totalDays = remaining?.months * 30 + remaining?.days;
+    if (totalDays > 180) {
+      return true;
+    }
+    return false;
+  };
+
+  entitlementRemainingAlert() {
+    return this.props.isLoggedIn ? (
```

Comment:
unit test, please

Reply:
Added in two new unit tests in `IntroductionPage.unit.spec.jsx`, one to check that the alert is displayed when remaining entitlement days are greater than 180 and another to check that the alert is not displayed when the remaining days are less than 180.